readme.md

```diff
@@ -103,6 +103,11 @@ using just `it` in `mapIt` and `filterIt` is just ... and makes code a little un
 (1..10) |> reduce((a1, a2, ...), acc = 2)
 (1..10) |> each(a1, a2)
 ```
+Custom idents work with both `op:` and `op()` style syntax:
+```nim
+(1..10) |> map: n => _
+(1..10) |> filter: (n, k) => _
```

Comment:
using `op: ...` style only is valid inside the `iterrr` macro.

Reply:
Ops. Copied and pasted without thinking. I will fix the readme.